packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1096,7 +1102,7 @@ export class SnapController extends BaseController<
   }
 
   _onOutboundRequest(snapId: SnapId) {
-    const runtime = this.#getRuntimeExpect(snapId);
+    const runtime = this.#getRuntimeExpect(snapId as ValidatedSnapId);
```

Comment:
I'm curious whether we should be asserting the validity of the snapId here? Or do you think casting is safe?

Reply:
I think casting here is safe, outbound requests would only be coming from an installed snap.